pkg/template/api/types.go

```diff
@@ -13,6 +13,9 @@ type Template struct {
 	unversioned.TypeMeta
 	kapi.ObjectMeta
 
+	// Optional: An instructional message for the user processing this template (parameter substitution will be performed).
+	PostProcessMessage string
```

Comment:
https://github.com/openshift/origin/blob/master/pkg/build/api/v1/types.go#L39-L68

my understanding is we're supposed to do it that way so the generated doc aligns with the json field names (which are also lower-camel-case).

@smarterclayton ?